driver/cam_hal.c

```diff
@@ -21,6 +21,7 @@
 #if (ESP_IDF_VERSION_MAJOR == 3) && (ESP_IDF_VERSION_MINOR == 3)
 #include "rom/ets_sys.h"
 #else
+#include "esp_timer.h"
```

Comment:
Please also add esp_timer component to component requirements for IDF 4.2+, as done here:

https://github.com/espressif/esp-modbus/blob/436bc70b3da58d1f376f6fe92ec0a6ad567b7493/CMakeLists.txt#L64-L67

Reply:
@igrr how to filter IDF version in CMakeLists?